view/frontend/web/js/view/payment/method-renderer/vindi-pix.js

```diff
@@ -4,19 +4,59 @@ define(
         'Magento_Checkout/js/view/payment/default',
         'mage/translate',
         'jquery',
-        'mageUtils'
+        'mageUtils',
+        'Vindi_Payment/js/model/document',
+        'Vindi_Payment/js/model/validate'
     ],
 
-    function (_, Component, $t, $, utils) {
+    function (_, Component, $t, $, utils, document, documentValidate) {
         'use strict';
         return Component.extend({
             defaults: {
-                template: 'Vindi_Payment/payment/vindi-pix'
+                template: 'Vindi_Payment/payment/vindi-pix',
+                document: document
             },
 
             getInfoMessage: function () {
                 return window?.checkoutConfig?.payment?.vindi_pix?.info_message;
-            }
+            },
+
+            isActiveDocument: function () {
+                return window?.checkoutConfig?.payment?.vindi_pix?.enabledDocument;
+            },
+
+            checkCpf: function () {
+                const message = !documentValidate.isValidCpf(this?.document?.value()) ? '' : 'CPF inválido';
```

Comment:
Acredito que aqui o correto seja `''` se verdadeiro e `'CPF inválido'` se falso não? 🤔 
```suggestion
                const message = documentValidate.isValidCpf(this?.document?.value()) ? '' : 'CPF inválido';
```

Reply:
@laerte-guimaraes você está certo, obrigado por ver esse detalhe.